src/main/java/uk/gov/hmcts/reform/et/syaapi/service/NotificationService.java

```diff
@@ -113,56 +115,202 @@ public SendEmailResponse sendSubmitCaseConfirmationEmail(CaseDetails caseDetails
+                    log.info("Respondent did not have an email address associated with their account");
+                } else {
+                    try {
+                        notificationClient.sendEmail(
+                            emailToRespondentTemplate,
+                            caseData.getClaimantType().getClaimantEmailAddress(),
+                            respondentParameters,
+                            caseId
+                        );
+                        log.info("Sent email to respondent");
+                    } catch (NotificationClientException ne) {
+                        throw new NotificationException(ne);
+                    }
+                }
+            });
     }
 
+    /**
+     * Format details of claimant request and retrieve case data, then send email to confirmation to tribunal.
+     *
+     * @param caseData        existing case details
+     * @param claimant        claimant's full name
+     * @param caseNumber      ethos case reference
+     * @param respondentNames concatenated respondent names
+     * @param hearingDate     date of the nearest hearing
+     * @param caseId          16 digit case id
+     */
+    public void sendAcknowledgementEmailToTribunal(CaseData caseData,
+                                                   String claimant,
+                                                   String caseNumber,
+                                                   String respondentNames,
+                                                   String hearingDate,
+                                                   String caseId,
+                                                   ClaimantTse claimantApplication
+    ) {
+
+        Map<String, Object> tribunalParameters = new ConcurrentHashMap<>();
+        addCommonParameters(
+            tribunalParameters,
+            claimant,
+            respondentNames,
+            caseId,
+            caseNumber
+        );
+        tribunalParameters.put(
+            HEARING_DATE,
+            hearingDate
+        );
+
+        String subjectLine =  caseNumber + " " + SHORT_TEXT_MAP.get(claimantApplication.getContactApplicationType());
+        tribunalParameters.put(
+            "subjectLine",
+            subjectLine
+        );
+
+
+        String managingOffice = caseData.getManagingOffice();
+        if (managingOffice.equals(UNASSIGNED_OFFICE) || isNullOrEmpty(managingOffice)) {
+            log.info("Could not send email as no office has been assigned");
+        } else {
+            try {
+                notificationClient.sendEmail(
+                    notificationsProperties.getTribunalAcknowledgementTemplateId(),
+                    caseData.getTribunalCorrespondenceEmail(),
+                    tribunalParameters,
+                    caseId
+                );
+            } catch (NotificationClientException ne) {
+                throw new NotificationException(ne);
+            }
+        }
+    }
```

Comment:
Similar to 225-233 lines? Or Im missing something, could we use one method for both places?